docs/content/docs/user-docs/irsa.md

```diff
@@ -86,96 +98,77 @@ read -r -d '' TRUST_RELATIONSHIP <<EOF
-```
-
-If you have not done so already, update the AWS region of your ACK service controller using the following commands:
-```bash
-# Update your controller with the correct AWS region. Example: us-east-1
-export AWS_SERVICE_NAME="s3"
-export ACK_K8S_RELEASE_NAME=ack-$AWS_SERVICE_NAME-controller
-kubectl -n $ACK_K8S_NAMESPACE set env deployment/$ACK_K8S_RELEASE_NAME \
-    AWS_REGION="$AWS_ACCOUNT_ID"
-```
-
-### Edit Helm charts directly on your local file system
-
-If you did not already install your ACK service controller using Helm charts, you can directly edit the Helm chart YAML files on your local file system. 
-
-Update the [`values.yaml` file][s3-helm-values] and set the value for `aws.region` and `serviceAccount.annotations`.
-
-```bash
-# Update variables as needed for your ACK service controller
-ACK_CONTROLLER_HELM_CHARTS_DIR=<directory containing Helm chart for ACK service controller>
-AWS_SERVICE_NAME="s3"
-ACK_K8S_NAMESPACE=ack-system
-ACK_K8S_RELEASE_NAME=ack-$AWS_SERVICE_NAME-controller
-
-cd "$ACK_CONTROLLER_HELM_CHARTS_DIR"
-
-# Dry run and verify the output
-helm install --debug --dry-run --namespace "$ACK_K8S_NAMESPACE" "$ACK_K8S_RELEASE_NAME" .
-
-# Install the ACK service controller on your cluster
-helm install --create-namespace --namespace "$ACK_K8S_NAMESPACE" "$ACK_K8S_RELEASE_NAME" .
+export IRSA_ROLE_ARN=eks.amazonaws.com/role-arn=$ACK_CONTROLLER_IAM_ROLE_ARN
+kubectl annotate serviceaccount -n $ACK_K8S_NAMESPACE $ACK_K8S_SERVICE_ACCOUNT_NAME $IRSA_ROLE_ARN
 ```
 
-Verify that the service account has been created on your cluster and that its annotations include the IAM role for your ACK service controller.
+Restart ACK service controller deployment using the following commands. The restart
+will update service controller pods with IRSA environment variables
 ```bash
-kubectl describe serviceaccount/$ACK_K8S_SERVICE_ACCOUNT_NAME -n $ACK_K8S_NAMESPACE
+# Note the deployment name for ACK service controller from following command
+kubectl get deployments -n $ACK_K8S_NAMESPACE
+kubectl -n $ACK_K8S_NAMESPACE rollout restart deployment <ACK deployment name>
 ```
 
-For detailed instructions, refer to Amazon EKS documentation on how to [associate an IAM role to a service account ][iam-service-account].
-
 ## Step 4: Verify successful setup
 
-When AWS clients or SDKs connect to an AWS API, they detect an [AssumeRoleWithWebIdentity][security-token] security token to assume the IAM role. 
+When AWS clients or SDKs connect to an AWS API, they detect an [AssumeRoleWithWebIdentity][security-token] security token to assume the IAM role.
 
 Verify that the `AWS_WEB_IDENTITY_TOKEN_FILE` and `AWS_ROLE_ARN` environment variables exist for your Kubernetes pod using the following commands:
 ```bash
-kubectl get pods -A
-kubectl exec -n $ACK_K8S_NAMESPACE <NAME> env | grep ack
+kubectl get pods -n $ACK_K8S_NAMESPACE
+kubectl exec -n $ACK_K8S_NAMESPACE <NAME> -- env | grep AWS
```

Comment:
I do not like this `exec` into the pod method. It is just as easy to describe the pod environment variables by describing the pod and grepping for them there: 
```suggestion
kubectl describe pod -n $ACK_K8S_NAMESPACE <NAME> | grep "^\s*AWS_"
```